src/main/java/com/icthh/xm/ms/balance/domain/BalanceChangeEvent.java

```diff
@@ -40,32 +43,52 @@
     @SequenceGenerator(name = "sequenceGenerator")
     private Long id;
 
-    private Long balanceId;
-    private String balanceKey;
-    private String balanceTypeKey;
-    private Long balanceEntityId;
-
-    @Column(precision = 10, scale = 2)
+    @NotNull
+    @Column(name = "amount_delta", precision = 10, scale = 2, nullable = false)
```

Comment:
nullable = false is missing in old version. Why you add this configuration?

Reply:
Because it's actually not nullable from liqubase scripts